src/components/Video/Video.scss

```diff
@@ -24,11 +23,14 @@
 		margin-top: 50px;
 		justify-content: center;
 	}
-
+	&__size {
```

Comment:
@ZohebMOPO , in BEM we don't name attributes with an underscore. Change it to something like `username-selector__input_container`